.golangci.example.yml

```diff
@@ -208,6 +208,10 @@ linters-settings:
   whitespace:
     multi-if: false
 
+  wsl:
+    # Skip test files
+    no-test: false
```

Comment:
1. please, don't use negative naming
2. please, don't add such duplicating option: any linter can be disabled for tests by `exclude-rules`. It's not performance effective, I agree, but it should be done in a general way for all linters, not per-linter

Reply:
Sorry about that, I'll remote this since the configuration isn't needed!